internal/cli/admincmd/install_client.go

```diff
@@ -628,3 +614,14 @@ func (c *InstallClient) validateSystemDomain() error {
 
 	return nil
 }
+
+func (c *InstallClient) getTraefikInfo(ctx context.Context) (string, error) {
+	s := c.ui.Progressf("Waiting for LoadBalancer IP on traefik service.")
+	if err := c.waitForTraefikIngressIP(ctx); err != nil {
+		s.Stop()
```

Comment:
Replace this and line 624 with `defer s.Stop()` between lines 619 and 620.

Reply:
oh true, we are now in a method, it will work nicely. Doing now.

Reply:
Fixed up here: https://github.com/epinio/epinio/pull/983/files#diff-34be35943d88eae52a69a908ea43cda8322c3b88cf152da75dd724ead2a333b2R620